### PR TITLE
refactor(Lists): FRMW-766 migrate Lists to element

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-766

### LGTMs
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `Lists` to an Element.

### CODE REVIEW

File: src/elements/Lists/lists.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Lists
```
Comment (100stacks) [inline src/elements/Lists/lists.less]: Pulled from `src/components/rxApp/common.less` into standalone LESS file.

**Comment by 100stacks** [comment]
## `Lists` Element

lists - encoreui 2016-04-25 14-03-47

**Comment by CITguy** [comment]
LGTM

**Comment by glynnis** [comment]
LGTM

**Pull request merged**